dashboards/Overview.json

```diff
@@ -63,576 +25,367 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": null,
-  "iteration": 1539648773344,
+  "id": 27,
+  "iteration": 1548397162679,
   "links": [],
   "panels": [
     {
-      "collapsed": false,
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "fill": 1,
       "gridPos": {
-        "h": 1,
-        "w": 24,
+        "h": 6,
+        "w": 15,
         "x": 0,
         "y": 0
       },
-      "id": 8,
-      "panels": [],
-      "title": "Components",
-      "type": "row"
-    },
-    {
-      "content": "# Admin Router Health\n\nplaceholder",
-      "gridPos": {
-        "h": 4,
-        "w": 4,
-        "x": 0,
-        "y": 1
-      },
-      "id": 10,
-      "links": [],
-      "mode": "markdown",
-      "title": "",
-      "transparent": false,
-      "type": "text"
-    },
-    {
-      "content": "# DC/OS Checks\n\nplaceholder",
-      "gridPos": {
-        "h": 4,
-        "w": 4,
-        "x": 4,
-        "y": 1
+      "id": 16,
+      "legend": {
+        "alignAsTable": true,
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "rightSide": true,
+        "show": true,
+        "total": false,
+        "values": false
       },
-      "id": 13,
+      "lines": true,
+      "linewidth": 1,
       "links": [],
-      "mode": "markdown",
-      "title": "DC/OS Checks",
-      "transparent": false,
-      "type": "text"
-    },
-    {
-      "content": "# Admin Router Health\n\nplaceholder",
-      "gridPos": {
-        "h": 4,
-        "w": 4,
-        "x": 8,
-        "y": 1
+      "nullPointMode": "null",
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "sum(mesos_master_frameworks_connected{dcos_cluster_name=\"$clustername\"})",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "Services",
+          "refId": "A"
+        },
+        {
+          "expr": "count(count by(task_name) (cpus_limit{dcos_cluster_name=\"$clustername\"}))",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "Tasks",
+          "refId": "B"
+        },
+        {
+          "expr": "count(count by(container_id) (cpus_limit{dcos_cluster_name=\"$clustername\"}))",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "Containers",
+          "refId": "C"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Running Containers",
```

Comment:
We should change this given this Graph has more information than just contianers?